test/metabase/pulse/render/test_util.clj

```diff
@@ -37,6 +37,7 @@
                                :highlight_row true}
                               {:columns       ["c"]
                                :type          "range"
+                               :operator      "="
```

Comment:
How is this related to the change?

Reply:
it's unrelated, I just add it for the sake of completeness. I believe `=` was the default operator and we could omit it.